orcid-persistence/src/test/resources/solr/org/conf/schema.xml

```diff
@@ -512,11 +512,11 @@
 			multiValued="false" />
 		<field name="org-disambiguated-name" type="orcid_text_name" indexed="true" stored="true"
 			multiValued="false" />
-		<field name="org-disambiguated-city" type="orcid_text_name" indexed="true" stored="true"
+		<field name="org-disambiguated-city" type="orcid_text_name" indexed="false" stored="true"
```

Comment:
@Camelia-Orcid, I think we should roll back all this schema.xml, because we dont want to use country, city nor region in the search typeahead; but, we could potentially want to use it in the future in queries done thorught the API's or directly by us in SOLR.